ontology/ontoavida.md

```diff
@@ -8,7 +8,9 @@ contact:
   github: miguelfortuna
   orcid: 0000-0002-8374-1941
 description: "OntoAvida develops an integrated vocabulary for the description of the most widely-used computational approach for studying evolution using digital organisms (i.e., self-replicating computer programs that evolve within a user-defined computational environment)."
-domain: digital evolution, a branch of Artificial Life.
+domain: other
```

Comment:
Maybe a stretch, but still better than having an "other" category
```suggestion
domain: organisms
```

Reply:
hmm.. I would prefer a new category "artificial life" rather than forcing that into "organisms", but I can be outvoted.

Reply:
Maybe simulation as a category. Includes cell simulations, for example, which we wouldn't call artificial life.